File: app/src/main/java/com/goranch/publicapis/api/model/food/nutrition/AltMeasure.java

package com.goranch.publicapis.api.model.food.nutrition;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AltMeasure {

    @SerializedName("serving_weight")
    @Expose
    private Double servingWeight;
    @SerializedName("measure")
    @Expose
    private String measure;
    @SerializedName("seq")
    @Expose
    private Integer seq;
    @SerializedName("qty")
    @Expose
    private Double qty;

    public Double getServingWeight() {
        return servingWeight;
    }

    public void setServingWeight(Double servingWeight) {
        this.servingWeight = servingWeight;
    }

    public String getMeasure() {
        return measure;
    }

    public void setMeasure(String measure) {
        this.measure = measure;
    }

    public Integer getSeq() {
        return seq;
    }

    public void setSeq(Integer seq) {
        this.seq = seq;
    }

    public Double getQty() {
        return qty;
    }

    public void setQty(Double qty) {
        this.qty = qty;
    }

}

File: app/src/main/java/com/goranch/publicapis/api/model/food/nutrition/AltMeasure.kt
package com.goranch.publicapis.api.model.food.nutrition

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AltMeasure {

    @SerializedName("serving_weight")
    @Expose
    var servingWeight: Double? = null
    @SerializedName("measure")
    @Expose
    var measure: String? = null
    @SerializedName("seq")
    @Expose
    var seq: Int? = null
    @SerializedName("qty")
    @Expose
    var qty: Double? = null

}
